src/app/Tools/Settings/Marketplace.tsx

```
@@ -0,0 +1,617 @@
+                        boxShadow: "0 10px 15px -3px rgba(0, 0, 0, 0.1), 0 4px 6px -2px rgba(0, 0, 0, 0.05)",
+                        outline: "2px solid var(--accent-color)"
+                    }}
+                >
+                    <CardContent className="p-4">
+                        <div className="flex items-start justify-between">
+                            <div className="flex items-center gap-3">
+                                <div className="w-10 h-10 flex items-center justify-center rounded-md overflow-hidden bg-gray-100 dark:bg-gray-800">
+                                    {bookmark.Icon && bookmark.isSVG ? (
+                                        <div
+                                            className="w-6 h-6"
+                                            dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)